### DISCUSSION TITLE
Status of missed 3.13 milestone items

### DISCUSSION THREAD

**Discussion by n2ygk** [encode/django-rest-framework#8327] [Question & Answer]
Anybody know what the plans are for the open items in the [Release 3.13 milestone](https://github.com/encode/django-rest-framework/milestone/77) that were skipped? Will some (any) of them get moved to the 3.14 milestone?

Asking for a friend.

;-)

**Comment by lovelydinosaur** [comment]
Well, there's a whole bunch of triage backlog that's going to need gradually addressing here.

That milestone is probably a reasonable indication of a handful of tickets that's worth us prioritising - do you have any items in particular that are important to you use cases?

I've bumped a "3.14" marker on 'em so we've at least got a handle on "we thought these ones looked worth prioritising at some point".

**Comment by n2ygk** [comment]
> do you have any items in particular that are important to you use cases?

#7516 to enable swagger-ui, etc. to log the user in with an autogenerated OpenAPI schema. I plan on adding downstream support in django-oauth-toolkit as well for oauth2 but this would establish the base of basic auth, sessions, etc.

**Reply by lovelydinosaur** [reply]
Okay, thanks - that's useful feedback.
